### DISCUSSION TITLE
QC: Address inconsistent file and function naming.

### DISCUSSION THREAD

**Discussion by samussiah** [Gilead-Public/gsm#502] [General]
## QC Details
The mix of camelCase, BigCamelCase, underscored, and dashed file names and functions might benefit from a consistent naming convention.  The [tidyverse style guide](https://style.tidyverse.org/files.html#names) seems to favor lowercase, underscored names while [Google's R style guide](https://google.github.io/styleguide/Rguide.html#naming-conventions) favors BigCamelCase.

Does Gilead have a style guide?  From this repo and others it seems BigCamelCase and Big_Underscore seem to be in vogue.

**Comment by samussiah** [comment]
Also consider `verb_domain()` instead of `domain_verb()`, e.g. `map_ae_raw`, `assess_ae`, etc.

**Comment by jwildfire** [comment]
I think this is fine to address as long as it isn't toooooooooo disruptive. 

@samussiah @mattroumaya can you all work together to create a list of proposed name changes and post them as a comment here.
